api-metadata.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "definitions": {
+
+    "function_metadata": {
+      "type": "object",
+      "properties": {
+        "minArgs": {
+          "description": "The minimum number of arguments which must be passed to the function. If called with fewer than this number of arguments, the wrapper will raise an exception.",
+          "type": "number"
+        },
+        "maxArgs": {
+          "description": "The maximum number of arguments which may be passed to the function. If called with more than this number of arguments, the wrapper will raise an exception.",
+          "type": "number"
+        },
+        "fallbackToNoCallback": {
+          "description": "If the function doesn't take a callback parameter in some browsers and throws an error when a callback is passed.",
+          "type": "boolean"
+        },
+        "noCallback": {
```

Comment:
Why is this here? Internal helper properties in the polyfill itself should not leak to the schema definition of the JSON file.

Reply:
If you remove the definition here, then validation of the JSON file with API metadata will (should) fail.

Reply:
I currently achieve that with: https://github.com/mozilla/webextension-polyfill/blob/34bdcfc63f63e03f55bf4f15181edafb19f55376/api-metadata.schema.json#L32-L36

But removing it should work too.